docs/current/quickstart/snippets/step4/dagger.mjs

```diff
@@ -0,0 +1,29 @@
+import { connect } from "@dagger.io/dagger"
+
+connect(async (client) => {
+
+  const source = await client.container()
+      .from("node:16")
+      .withMountedDirectory('/src', client.host().directory('.'))
+      .withExec(['git', 'clone', 'https://github.com/vikram-dagger/hello-dagger'])
+      .id()
+
+  const runner = client
+    .container({ id: source })
+    .withWorkdir("./hello-dagger")
+    .withExec(["npm", "install"])
+
+   // run tests
+  await runner
+    .withExec(["npm", "test", "--", "--watchAll=false"])
+    .exitCode()
+
+  // TODO: check exit code and do next steps only if all tests pass or dump error and exit
```

Comment:
We don't need to check any status code here because the pipeline will exit with an error if tests don't pass. That's the current behavior of the SDK.